lang/cs/Org.Apache.REEF.Tang/Annotations/NamedParameter.cs

```diff
@@ -30,9 +31,11 @@ public class NamedParameterAttribute : System.Attribute
         public Type DefaultClass { get; set; }
         public string[] DefaultValues { get; set; }
         public Type[] DefaultClasses { get; set; }
+        public string Alias { get; set; }
+        public string AliasLanguage { get; set; }
 
         public NamedParameterAttribute(string documentation = "", string shortName = "",
```

Comment:
This is a `public` constructor. We shouldn't change it without first making it `Obsolete`. Maybe we can add a new `internal` one and make the old one `Obsolete`?